### PR TITLE
Use company details from supplier for agreement details view

### PR DESCRIPTION
**Pull request by lfdebrux**
Trello: https://trello.com/c/zBkCpdEs/594-wrong-supplier-registered-details-showing-in-admin-framework-agreement-page

### CODE REVIEW

File: app/main/helpers/supplier_details.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def get_company_details_from_supplier(supplier):
             or
             supplier.get("otherCompanyRegistrationNumber")
         ),
+        "registered_with": "companies_house" if "companiesHouseNumber" in supplier else "unknown",
```
Comment (katstevens) [inline app/main/helpers/supplier_details.py]: Where do we use this?

Comment (lfdebrux) [reply app/main/helpers/supplier_details.py]: Oh I was going to add a conditional to the Companies House URL but I forgot to add that bit.

Comment (lfdebrux) [reply app/main/helpers/supplier_details.py]: Added now

File: example_responses/supplier_response.json [hunk 2]
```diff
@@ -21,14 +20,24 @@
     "description": "Supplier Description",
     "dunsNumber": "1234",
     "eSourcingId": "1234",
+    "companiesHouseNumber": "1234578",
+    "registrationCompany": "country:GB",
+    "tradingStatus": "limited company (LTD)",
+    "organisationSize": "medium",
+    "VAT Number": "GB 123 4567 89",
```
Comment (katstevens) [inline example_responses/supplier_response.json]: `VAT Number`? That's long gone 🤔

Comment (lfdebrux) [reply example_responses/supplier_response.json]: Hmm, that's funny, it was in a response I was looking at... I guess the supplier was created a while ago. Shall I remove?

**Comment by lfdebrux** [comment]
> WFM - could maybe do with a test case that covers the non-companies-house path but otherwise 👍

Added

**Comment by risicle** [comment]
Woop woop woop

**Pull request merged**